demo_drivers/meshing/mesh_from_gmsh/mesh_from_gmsh_solid.cc

```diff
@@ -0,0 +1,273 @@
+//
+// Created by iqraa on 13-9-24.
+//
+//LIC// ====================================================================
+//LIC// This file forms part of oomph-lib, the object-oriented,
+//LIC// multi-physics finite-element library, available
+//LIC// at http://www.oomph-lib.org.
+//LIC//
+//LIC// Copyright (C) 2006-2023 Matthias Heil and Andrew Hazel
+//LIC//
+//LIC// This library is free software; you can redistribute it and/or
+//LIC// modify it under the terms of the GNU Lesser General Public
+//LIC// License as published by the Free Software Foundation; either
+//LIC// version 2.1 of the License, or (at your option) any later version.
+//LIC//
+//LIC// This library is distributed in the hope that it will be useful,
+//LIC// but WITHOUT ANY WARRANTY; without even the implied warranty of
+//LIC// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+//LIC// Lesser General Public License for more details.
+//LIC//
+//LIC// You should have received a copy of the GNU Lesser General Public
+//LIC// License along with this library; if not, write to the Free Software
+//LIC// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+//LIC// 02110-1301  USA.
+//LIC//
+//LIC// The authors may be contacted at oomph-lib@maths.man.ac.uk.
+//LIC//
+//LIC//====================================================================
+
+//Oomph-lib includes
+#include "generic.h"
+#include "solid.h"
+#include "constitutive.h"
+
+// The mesh form gmsh
+#include "meshes/gmsh_brick_mesh.h"
+
+using namespace std;
+using namespace oomph;
+
+
+
+/// ////////////////////////////////////////////////////////////////////
+/// ////////////////////////////////////////////////////////////////////
+/// ////////////////////////////////////////////////////////////////////
+
+
+//=======start_namespace==========================================
+/// Global variables
+//================================================================
+namespace Input
+{
+    /// Elastic modulus
+    double E=1.0e8;
+
+    /// Poisson's ratio
+    double Nu=0.3;
+
+
+    /// Pointer to constitutive law
+    ConstitutiveLaw* Constitutive_law_pt= nullptr;
+    double density = 2500;
+    /// Non-dim gravity
+    double Gravity=9.81;
+
+    /// Non-dimensional gravity as body force
+    void body_force(const double& time,
+                    const Vector<double> &xi,
+                    Vector<double> &b)
+    {
+        b[0]=0.0;
+        b[1]=0.0;
+        b[2]=-Gravity*density;
+    }
+
+
+}
+
+
+
+template<class ELEMENT>
+class CantileverProblem : public Problem
+{
+
+public:
+
+    /// Constructor:
+    CantileverProblem(const string& mesh_file)
+    {
+
+        Problem::mesh_pt() = new SolidGmshBrickMesh<ELEMENT>(mesh_file, false);
```

Comment:
comment, especially on the false (which should be defined as a meaningful variable!)